driver-core/src/test/unit/com/mongodb/client/model/FindOneAndUpdateOptionsSpecification.groovy

```diff
@@ -115,21 +115,4 @@ class FindOneAndUpdateOptionsSpecification extends Specification {
         where:
         arrayFilters << [null, [], [new BsonDocument('a.b', new BsonInt32(1))]]
     }
-
-
-    def 'should set hint'() {
-        expect:
-        new UpdateOptions().hint(hint).getHint() == hint
```

Comment:
It looks like we should have these tests, but they should be like:

     new FindOneAndUpdateOptions().hint(hint).getHint() == hint